portfolio/src/main/webapp/gallery.html

```diff
@@ -32,122 +29,149 @@
             </div>
         </nav>
         <div class="w-100 title">
-            <h1 class = "display-3">Welcome to my gallery!</h1>
+            <!-- <h1 class = "display-3">WELCOME TO MY GALLERY!</h1> -->
+            <svg class="intro go" viewbox="0 0 300 50">
+                <text text-anchor="start" x="10" y="30" class="text text-stroke" clip-path="url(#text1)">WELCOME</text>
+                <text text-anchor="start" x="123" y="30" class="text text-stroke" clip-path="url(#text2)">TO</text>
+                <text text-anchor="start" x="160" y="30" class="text text-stroke" clip-path="url(#text3)">MY</text>
+                <text text-anchor="start" x="200" y="30" class="text text-stroke" clip-path="url(#text4)">GALLERY</text>
+                <text text-anchor="start" x="10" y="30" class="text text-stroke text-stroke-2" clip-path="url(#text1)">WELCOME</text>
+                <text text-anchor="start" x="123" y="30" class="text text-stroke text-stroke-2" clip-path="url(#text2)">TO</text>
+                <text text-anchor="start" x="160" y="30" class="text text-stroke text-stroke-2" clip-path="url(#text3)">MY</text>
+                <text text-anchor="start" x="200" y="30" class="text text-stroke text-stroke-2" clip-path="url(#text4)">GALLERY</text>
+                <defs>
+                    <clipPath id="text1">
+                    <text text-anchor="start" x="10" y="30" class="text">WELCOME</text>
+                    </clipPath>
+                    <clipPath id="text2">
+                    <text text-anchor="start" x="123" y="30" class="text">TO</text>
+                    </clipPath>
+                    <clipPath id="text3">
+                    <text text-anchor="start" x="160" y="30" class="text">MY</text>
+                    </clipPath>
+                <clipPath id="text4">
+                    <text text-anchor="start" x="200" y="30" class="text">GALLERY</text>
+                    </clipPath>
+                </defs>
+            </svg>
+<!-- <div></div> -->
```

Comment:
Remove?

Reply:
I''m not sure if I can separate them out, since I create this branch based on background branch. But I can merge the two branch once they are both reviewed.